### PR TITLE
Add yaml semantic conventions for RPC

### PR DESCRIPTION
**Pull request by thisthat**
## Changes

Updated RPC semantic convention to the YAML format.

Related issues #547

### CODE REVIEW

**Comment by Oberon00** [comment]
#547 is closed. Maybe you want to open a new tracking issue for these?

File: specification/trace/semantic_conventions/rpc.md [hunk 1]
```diff
@@ -48,17 +48,23 @@ Examples of span names:
...
-| `net.peer.port` | See [network attributes][]. | See below |
-| `net.transport` | See [network attributes][]. | See below |
-
-At least one of [network attributes][] `net.peer.name` or `net.peer.ip` is required.
+<!-- semconv rpc -->
+| Attribute  | Type | Description  | Example  | Required |
+|---|---|---|---|---|
+| `rpc.system` | string | A string identifying the remoting system. | `grpc`<br>`java_rmi`<br>`wcf` | Yes |
+| `rpc.service` | string | The full name of the service being called, including its package name, if applicable. | `myservice.EchoService` | Conditional<br>No, but recommended |
+| `rpc.method` | string | The name of the method being called, must be equal to the $method part in the span name. | `exampleMethod` | Conditional<br>No, but recommended |
```
Comment (Oberon00) [inline specification/trace/semantic_conventions/rpc.md:56]: "Condiditional. No" So it's not conditionally required? That sounds a bit odd. Maybe the tool should just not emit "conditional" but just put the note in? Or there should be an additional level between not required and required named "recommended". Or this should be just not required.

Comment (thisthat) [reply specification/trace/semantic_conventions/rpc.md:56]: I also noticed this problem. Imho the tool should be updated to only emit the specified text without the `Conditional` prefix.

Comment (Oberon00) [reply specification/trace/semantic_conventions/rpc.md:56]: That sounds also fine, although it might lead to mis-use. Like "Required: Yes, because bla"

Comment (tigrannajaryan) [reply specification/trace/semantic_conventions/rpc.md:56]: +1 to what @Oberon00 said. I am not sure how to read the "Conditional No, but recommended" phrase.

Comment (arminru) [reply specification/trace/semantic_conventions/rpc.md:56]: I filed https://github.com/open-telemetry/build-tools/issues/7 for having this fixed in the MD generator. @thisthat will work on this soon so I would be fine with merging this PR and re-generate the updated markdown files in a follow-up.

**Review by arminru — APPROVED** [review]

**Review by tigrannajaryan — APPROVED** [review]
LGTM, given that the formatting will be fixed by https://github.com/open-telemetry/build-tools/issues/7

**Review by Oberon00 — APPROVED** [review]

**Comment by thisthat** [comment]
@open-telemetry/specs-approvers please have a look

**Review by carlosalberto — APPROVED** [review]

**Pull request merged**